### ISSUE TITLE
Lỗi invalid literal for int() with base 10: ''

### ISSUE THREAD

**Issue by tbuikr** [aivivn/aivivn_issues#8]
Chào ban tổ chức,

Mình thử gửi một file lên hệ thống nhưng bị lỗi. Lỗi mình như sau "invalid literal for int() with base 10:" 
Còn đây là code mình dùng
```
    pred_output =[]
    filename_pred =[]
    k =5 # Lay 5 kq tot nhat
    with torch.no_grad():
        for data, target,path in test_loader:
            data, target, path = data.to(device), target, path
            output = model(data)
            output = np.argsort(output.data.cpu().numpy(), axis = 1)[:, -k:][:, ::-1]    
            filename_pred.append(path[0].strip())
            pred_output.append(str(output[0])[1:-1])
  
    import pandas as pd
    sample_sub_all = pd.DataFrame(columns=["image", "label"])
    for filename, pred in zip(filename_pred , pred_output):
        sample_sub_all = sample_sub_all.append({
            "image": filename,
            "label": pred
        }, ignore_index=True )
    sample_sub_all.to_csv("test_submission.csv", index=False)
```

Mình đầu tiên predict từng image một và lấy output lưu vào một list. Sau đó, dùng pandas để ghi ra file csv (như đính kèm). Khi submit thì bị lỗi như trên. Phiền ban tổ chức xem và fix nha 

[test_submission.csv.zip](https://github.com/aivivn/aivivn_issues/files/2979677/test_submission.csv.zip)

**Comment by tiepvupsu** [comment]
Mình vừa kiểm tra file của bạn, là do bạn để thừa dấu cách sau dấu phẩy:
```
00089bdb1adf49a6988f54dbc5d7ef30.png,354 518 203 726 277
0008cf21d190404e9202b65b7f7b1510.png,746   3 167 153 296
000ab9d805bc4caeb1d8fe9bcd752602.png,175 410 311 155 590
000cbb4c85ab49c2bee1d67809886dcf.png,818 537  52 809 888
000cca1b5c834634a9c4fc32c8932fbf.png,383 197 553 363  85
002224d45a38429db04b1dff5be0924a.png, 55  42 194 137 509
```
dòng cuối cùng trước số 55 có một dấu cách. Bạn thử lưu ra file với chỉ một dấu cách giữa các label và không dấu cách sau dấu phẩy xem.

**Comment by tiepvupsu** [comment]
Bên mình sẽ fix lỗi có nhiều dấu cách sớm.

**Comment by tbuikr** [comment]
Mình vừa viết lại code để fix lỗi đó. Thanks. 
>Chú ý: các đáp án cách nhau bởi một dấu cách, mỗi đáp án là một số tự nhiên từ 0 đến 1000. Trong đó nhãn 1000 được đưa ra nếu đội chơi cho rằng bức ảnh đó nằm ngoài các nhãn trong tập huấn luyện.

**Comment by tiepvupsu** [comment]
Cảm ơn bạn.
